### DISCUSSION TITLE
[脚本无关] B站的相同清晰度不同播放策略的实际效果差距好像比想象的大

### DISCUSSION THREAD

**Discussion by CKylinMC** [the1812/Bilibili-Evolved#3268] [综合讨论 / General]
也是今天看视频的时候发现的，我还以为我熬夜眼花了。

我默认是AVC的观看，今天打开视频就被马赛克糊脸了，换成HEVC的话就没有这样的问题。

1080P普通，HEVC和AVC对比：

![D 0%90SITCI%}{NK789B_5P](https://user-images.githubusercontent.com/11648014/164275893-108272da-116f-4e5f-8b57-6f14718863c8.png)

![IS`ISM6 SY_LP O%SZXNQW6](https://user-images.githubusercontent.com/11648014/164275928-d4ccfeca-089d-4947-98f8-2426fed7f65a.png)

如果看不清楚的话，缩放×432%，看看脸

![){}9SK{90C_MOMLRT%U`2N1](https://user-images.githubusercontent.com/11648014/164276023-0d35eebe-9589-45b3-8681-b37582418708.png)

![M$6ZLW4Z(X PSA@GQUK{FFU](https://user-images.githubusercontent.com/11648014/164276043-1dfc6b8c-4b3b-47db-b7ab-66a8adb6143b.png)

还是挺明显的。

视频码率差距也挺大，1242 Kbps 对比 955 Kbps

我不太了解视频编码究竟对最终画质表现有多少影响，但是相同画质仅仅只是播放策略差异就差了这么多，就...挺惊讶的

**Comment by the1812** [comment]
这个主要还是码率区别吧, 我猜高码率的 AVC 一样能比低码率的 HEVC 看起来更清晰
HEVC 主要还是用更多的性能消耗换取更低的带宽

**Reply by CKylinMC** [reply]
确实码率差距，但是自称相同清晰度只是换了一个很多用户不明白也不知道的选项就差了这么多我觉得离谱
